policy/commit/signature.rego

```diff
@@ -1,4 +1,4 @@
-package policy.release.commit_signature
+package policy.commit.signature
```

Comment:
Why not just move this into `policy/release`? I can see an argument for a commit being a property of a release (or image).